frontend/sass/_issues.scss

```diff
@@ -28,17 +28,20 @@
     }
 }
 
+.jf-issue-area-link:focus {
+    @include outline-focus();
+}
+
 .jf-issue-area-link:not(.jf-issue-count-zero) {
     background-color: $primary;
-    color: #fff;
-}
+    color: $primary-invert;
```

Comment:
@romeboards just FYI this is the preferred Bulma way to specify the complement/inverse of a Bulma-defined color, rather than e.g. hard-coding white or black.  Aside from being more readable (IMO) it also means that if we ever decided to e.g. change the primary color to be lighter, we wouldn't then have to manually change bunch of `#fff`s to `#000`s.

Another neat thing is that Bulma comes with a helper mixin, [`findColorInvert()`](https://bulma.io/documentation/overview/functions/), which can find the inverts for us so we don't even have to manually figure those out all the time!